vita3k/host/include/host/state.h

```diff
@@ -79,6 +79,7 @@ struct HostState {
     uint32_t fps = 0;
     uint32_t ms_per_frame = 0;
     WindowPtr window = WindowPtr(nullptr, nullptr);
+    CPUBackend cpu_backend;
```

Comment:
No please don't do this

Reply:
why ? i need this

Reply:
> 
> 
> Just use host.kernel.cpu_backend

oh hmm i check sorry

Reply:
fixed sorry, i have don't see this value exist